### PR TITLE
Version Bump

### PR DESCRIPTION
**Pull request by AmintorDusko**
Updating changelog and _version.py for v0.29-dev.

### CODE REVIEW

**Comment by multiphaseCFD** [comment]
Nice work Amintor! Just corrected some previous typo errors. Please double checked it.

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -60,8 +79,6 @@ and change `lightning.gpu` to inherit from `QubitDevice` instead of `LightningQu
 * Implement improved `stopping_condition` method, and make Linux wheel builds more performant.
 [(#77)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/77)
```
Comment (multiphaseCFD) [inline .github/CHANGELOG.md:81]: Do we have to remove this? It's kept in previous release changelog.

Comment (AmintorDusko) [reply .github/CHANGELOG.md:81]: This is the previous changelog.

Comment (AmintorDusko) [reply .github/CHANGELOG.md:81]: Sorry, I don't know if I got your point.

Comment (AmintorDusko) [reply .github/CHANGELOG.md:81]: Do you mean the `### Documentation` tags?

Comment (multiphaseCFD) [reply .github/CHANGELOG.md:81]: yes, just make it consistent.

**Review by multiphaseCFD — APPROVED** [review]

**Review by chaeyeunpark — APPROVED** [review]
Looks good to me!

**Pull request merged**